python/common/python/time.java

```diff
@@ -89,10 +89,34 @@ public time() {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "ctime(seconds) -> string\n\n" +
+                      "Convert a time in seconds since the Epoch to a string in local time.\n" +
+                      "This is equivalent to asctime(localtime(seconds)). When the time tuple is\n" +
+                      "not present, current time as returned by localtime() is used.",
+            default_args = {"seconds"}
     )
-    public static org.python.Object ctime() {
-        throw new org.python.exceptions.NotImplementedError("time.ctime() has not been implemented.");
+    public static org.python.Object ctime(org.python.Object seconds) {
+        java.util.Date date;
+        if (seconds == null) {
+            long currentTimeInMillis = System.currentTimeMillis();
+            date = new java.util.Date(currentTimeInMillis);
+        } else {
+            //Type checking the parameter
+            if (!(seconds instanceof org.python.types.Int) && !(seconds instanceof org.python.types.Float)) {
```

Comment:
I see this is testing for float but we're not exercising it in the tests. Can you add a test passing a float as well?

Reply:
I added another float test case too. BTW [print(time.ctime((now - (now % 3600)))](https://github.com/pybee/voc/pull/739/files#diff-3e3a3c1d82261ce04faab17d5b8024c0R121) already had a float test.